src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -279,8 +339,9 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
       ttl,
       certificates,
       withdrawals,
+      null,
```

Comment:
does this mean we don't support transaction metadata at all and hardcode it to null?

Reply:
Seems like that's the case.